tests/BuildScriptGenerator.Tests/Php/PhpLanguageDetectorTest.cs

```diff
@@ -95,15 +93,13 @@ public void Detect_ReturnsVersionFromCliSwitch_EvenIfEnvironmentVariable_AndComp
         }
 
         [Fact]
-        public void Detect_ReturnsVersion_FromEnvironmentVariable_EvenIfComposerFileHasVersionSpecified()
+        public void Detect_ReturnsVersion_FromOptions_EvenIfComposerFileHasVersionSpecified()
         {
             // Arrange
-            var environment = new TestEnvironment();
-            environment.Variables[PhpConstants.PhpRuntimeVersionEnvVarName] = "7.2.5";
             var detector = CreatePhpLanguageDetector(
                 supportedPhpVersions: new[] { "7.3.14", "7.2.5", "5.6.0" },
                 defaultVersion: "7.3.14",
-                environment);
+                new PhpScriptGeneratorOptions { PhpVersion = "7.2.5" });
             var repo = new MemorySourceRepo();
             var version = "5.6.0";
             repo.AddFile("{\"require\":{\"php\":\"" + version + "\"}}", PhpConstants.ComposerFileName);
```

Comment:
nit: maybe its just better to have a sample file instead of having all these string operations multiple times

Reply:
Not really related to this PR's change, so I am not going to fix this. I am resisting the temptation to do changes not related to this PR's intention :-)